### PR TITLE
Add mass of zombie spiders

### PR DESCRIPTION
**Pull request by LyleSY**
Adds a mass of zombie spiders.

#### Summary

SUMMARY: Content "Adds new monster 'mass of zombie spiders'"

#### Purpose of change

Add monster variety in midgame 

#### Describe the solution

New monster combining traits of other zombie animals, the blob, and existing spiders, spawns in forests and swamps on a timer, arriving around midgame

#### Describe alternatives you've considered

Jumping powers, but that doesn't fit the theme, better venom but I wanted this to be more generic

#### Testing

Monster spawns, behaves as expected, no errors
Screen Shot 2020-06-02 at 9 25 15 PM
Travis errors don't appear to be related

#### Additional context

N/A

### CODE REVIEW

**Comment by ymber** [comment]
Why would a 23kg mass of spiders end up stuck together?

**Comment by LyleSY** [comment]
> Why would a 23kg mass of spiders end up stuck together?

I've been thinking about that too. The game mechanical reason is that the blob can't infect anything less than that weight (I don't know why), so it is that weight. So a blob or something blob-infected went through the forest and swamp gathering spiders to build this. I like a jabberwock for this task, nice creepy project for a jabberwock to focus on, makes a good mental image. Once there were enough spiders to play, the jabberwock "bit" them.

**Comment by Fris0uman** [comment]
I'm not sure jabberwok is part of the zombie/blob faction, I think it's an independant thing, no?

**Comment by LyleSY** [comment]
> I'm not sure jabberwok is part of the zombie/blob faction, I think it's an independant thing, no?

Hey, you're right. I'd never checked the code on that, but I now recall I've kited it into zombies before on day 1. Yep, it's made of zombie flesh but it kills zombies. I don't know what the deal is with that. Fair enough, another blob-aligned creature then, either existing in game or yet to be created.

File: data/json/monsters/zed-animal.json [hunk 1]
```diff
@@ -376,5 +376,51 @@
       "REVIVES",
       "FILTHY"
     ]
+  },
+  {
+    "id": "mon_zpider_mass",
+    "type": "MONSTER",
+    "name": { "str": "mass of zombie spiders" },
+    "description": "Dozens, hundreds of spiders piling up high, each slowly oozing sticky green pus, struggling to keep the fetid mass together and moving.",
```
Comment (Fris0uman) [inline data/json/monsters/zed-animal.json:384]: 23 kg of spider is much much more than hundreds of them ^^"

Comment (meladath) [reply data/json/monsters/zed-animal.json:384]: Yeah, can this be changed to something like "thousands, maybe even millions", the heaviest spider irl is ~170g, so its thousands at minimum really.

**Comment by LyleSY** [comment]
The Travis errors appear to be from some modded content called "goblin_palette"

**Comment by Brian-Otten** [comment]
Just wanted to pitch in to say i quite like this. we've already seen the blob using collected bodyparts to make larger forms in the dissoluted devourer, and with this new creature it's shown off that if given time the blob is perfectly willing to consume all the biomass, not just that of larger creatures.

**Comment by GGgatherer** [comment]
If it consists of thousands of small (zombie)spiders, it should be effectively immune to any damage except bashing. Maybe change it to "dozen(s) of big mutated spiders"? "big" because we already have huge dog/human-sized spiders both in vanilla and Magyclism.

**Comment by Mister-Cultist** [comment]
Jesus Christ how horrifying

**Pull request merged**